react_practice/react-apps/waitlist-page-v1/src/CTAForm.css

```diff
@@ -0,0 +1,58 @@
+.cta-form-btn {
+  display: block;
+  width: 50%;
+  margin: 10px auto 0;
+}
+
+.cta-form {
+  position: absolute;
+  left: 50%;
+  max-width: 35%;
+  background-color: #fff;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.15), 0 2px 2px rgba(0, 0, 0, 0.15),
+    0 4px 4px rgba(0, 0, 0, 0.15), 0 8px 8px rgba(0, 0, 0, 0.15);
+  text-align: center;
+  border-radius: 15px;
+}
+
+.cta-join-header {
+  font-size: 50px;
+  color: #333;
+}
+
+.cta-description {
+  max-width: 80%;
+  margin: 0 auto;
+}
+
+.cta-description span {
+  color: #1e90ff;
+}
+
+.cta-form p {
+  max-width: 60%;
+  margin: 0 auto;
+  font-weight: 500;
+  line-height: 1.7em;
+}
+
+.cta-search {
+  margin: 10px auto 0;
+}
+.cta-search span {
+  color: #1e90ff;
+}
+
+.cta-shop span {
+  color: #1e90ff;
+}
+
+.cta-save span {
+  color: #1e90ff;
+}
+
+.cta-form-email {
+  padding: 10px 0 10px 5px;
+  margin-top: 50px;
+  border-radius: 5px;
+}
```

Comment:
All of this was brought over from `Home.css`. The reason I moved this here is that these are all styles specific to the `CTAForm.js` component. Having these styles sitting right next to the component that style them really help with organization of a project as you're not needing to try and remember where you put these styles.

As a project grows, you'll find it hard to remember where to find things, so you really want to try to optimize for that point. A little bit of organization up front can go a long way in long-term maintenance of a project. I find this particular bit of organization scales well with a codebase.

Note that this is only a convention. The styles were included regardless of whether they were defined in this file or in `Home.css`.